site/_scss/site/objects/_mixins.scss

```diff
@@ -0,0 +1,14 @@
+@mixin breakpoint($point) {
```

Comment:
@amymanion do you need this? because there is a built in bootstrap mixin you could use instead:

```
@include media-breakpoint-up($size) { ... }
```